tools/spec2vec/spec2vec_training.xml

```diff
@@ -0,0 +1,254 @@
+        sh ${weights_symlink} &&
+        sh ${spec2vec_python_cli}
+    ]]></command>
+        
+    <configfiles>
+        <configfile name="weights_symlink">
+            ln -fs '${weights_filename}' '${weights_filename}.npy'
+        </configfile>
+        <configfile name="spec2vec_python_cli">
+            python3 ${__tool_directory__}/spec2vec_training_wrapper.py \
+            --spectra_filename '$spectra_filename' \
+            --spectra_fileformat '$spectra_filename.ext' \
+            #if $output_parameters.model_checkpoints.is_true
+            --checkpoints '$output_parameters.model_checkpoints.checkpoints' \
+            #else
+            --epochs $output_parameters.model_checkpoints.epochs \
+            #end if
+            --vector_size $training_parameters.vector_size \
+            --alpha $training_parameters.alpha \
+            --min_alpha $training_parameters.min_alpha \
+            --window $training_parameters.window \
+            --min_count $training_parameters.min_count \
+            --sample $training_parameters.sample \
+            --seed $training_parameters.seed \
+            --sg $training_parameters.sg_param.sg \
+            #if not $training_parameters.sg_param.sg
+            --cbow_mean $training_parameters.sg_param.cbow_mean \
+            #end if
+            --hs $training_parameters.hs_param.hs \
+            #if not $training_parameters.hs_param.hs
+            --negative $training_parameters.hs_param.negative \
+            --ns_exponent $training_parameters.hs_param.ns_exponent \
+            #end if
+            --sorted_vocab $training_parameters.sorted_vocab \
+            --batch_words $training_parameters.batch_words \
+            --shrink_windows $training_parameters.shrink_windows \
+            #if $training_parameters.trim_vocab.max_vocab_size_bool
+            --max_vocab_size $training_parameters.trim_vocab.max_vocab_size \
+            #end if
+            --n_decimals $training_parameters.n_decimals \
+            --n_workers \${GALAXY_SLOTS:-1} \
+            #if $output_parameters.as_pickle
+            --model_filename_pickle '$model_filename_pickle' \
+            #end if
+            --model_filename '$model_filename' \
+            --weights_filename '$weights_filename' \
+        </configfile>
+    </configfiles>
+
+    <inputs>
+        <param label="Training spectra" name="spectra_filename" type="data" format="msp,mgf"
+               help="Spectra file to train a Spec2Vec model."/>
+
+        <section title="Output parameters" name="output_parameters" expanded="true">
+            <param label="Save model as Pickle file" name="as_pickle" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE"
+                   help="Add a Pickle output besides default JSON."/>
+            <conditional name="model_checkpoints">
+                <param label="Model checkpoints" name="is_true" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE"
```

Comment:
The IUC guidelines recommend using a select field here: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#booleans

Reply:
Thank you for the suggestion! I wasn't aware of that. Switched the input to `select` in 0cc4fa0.